DocumentedExamples/IcePlottingExample.ipynb

```diff
@@ -4,11 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<p>I think using mom5 instead of cice and mom6 instead of sis is a bit more intuitive as people might not know how the sea ice model is called, especially if they are oceanographers. Then you can add the sea ice model name in the model_args dictionary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I actually think the opposite ... the mom5 run uses mom5+cice5, the mom6 run uses mom6+sis2 but in the future ACCESS-OM3 will use mom6+cice6. So labelling by the sea-ice model is clearer. In the future we will have mom6 results where these variable names will need to come from cice6, not sis.</p>